openmmtools/mcmc.py

```diff
@@ -294,6 +355,19 @@ def minimize(self, tolerance=1.0*unit.kilocalories_per_mole/unit.angstroms,
 
         #timer.report_timing()
 
+    def _create_context_caches(self):
```

Comment:
We wouldn't need this method either, since we would just either use the provided `context_cache` in `run()` or fall back to the `cache.global_context_cache`.

Reply:
As discussed, this method is useful to have a centralized point where to handle the context caches behavior. This method is accesed from `create()` and `from_storage()`, so it reduces redundant code.

Reply:
OK!